packages/shared/components/Pane.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+    import { Height, Overflow } from './enums'
+
+    export let overflow: Overflow = Overflow.Hidden
+    export let height: number | Height = Height.Auto
```

Comment:
Why is `number` an option? It's not being used anywhere and  `height={13}` wouldn't work. To me `height` would be more of a px thing than multiples of 4. Also, if a specific height is needed this could be a solution: 

```svelte
<div class="h-12">
    <Pane height={Height.Full}>
    // ...
</div>

Reply:
debris, I change it :pray: